bond/CMakeLists.txt

```diff
@@ -1,14 +1,33 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
+
 project(bond)
 
-find_package(catkin REQUIRED message_generation std_msgs)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
 
-add_message_files(DIRECTORY msg
-  FILES
-  Constants.msg
-  Status.msg
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "msg/Constants.msg"
+  "msg/Status.msg"
+  DEPENDENCIES std_msgs builtin_interfaces
```

Comment:
Also `ADD_LINTER_TESTS`

Reply:
@mjcarroll , Linter tests are already added in this CMakeList